key.core/src/main/java/de/uka/ilkd/key/proof/init/DependencyContractPO.java

```diff
@@ -304,8 +304,7 @@ public void fillSaveProperties(Properties properties) {
      * @return The instantiated proof obligation.
      * @throws IOException Occurred Exception.
```

Comment:
```suggestion
```
Same for the other changes.

Reply:
Thanks for the review! Should be resolved with the new commit (hope I did not overlook one).